gnocchi/tests/functional/gabbits/resource-type.yaml

```diff
@@ -819,7 +819,7 @@ tests:
           path: /attributes/unknown
       status: 400
       response_strings:
-       - "can't remove non-existent object 'unknown'"
+       - can't remove non-existent object 'unknown'
```

Comment:
same here:

The word `a` is missing from this message. See the test failure for the exact message, but I think it's supposed to be "can't remove _a_ non-existent..."